workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/RenderWorkflowInTest.kt

```diff
@@ -93,7 +93,7 @@ class RenderWorkflowInTest {
   }
 
   @Test
-  fun side_effects_from_initial_rendering_in_root_workflow_are_never_started_when_scope_cancelled_before_start() {
+  fun `side_effects_from_initial_rendering_in_root_workflow_are_never_started_when_scope_cancelled_before_start`() {
```

Comment:
So we can use backticks just not spaces? was this becasue ktlint was failing these without the backticks?

Reply:
yes precisely, we can use backticks but no special characters when compiling to JS.
ktlint marks this as a MAX_LENGTH violation without the backtick